packages/sit-onyx/src/components/OnyxBottomBar/types.ts

```diff
@@ -0,0 +1,8 @@
+import type { DensityProp } from "../../composables/density";
+
+export type OnyxBottomBarProps = DensityProp & {
+  /**
+   * Whether to remove the top border / separator.
+   */
+  hideTopBorder?: boolean;
```

Comment:
Since we only have one border we could also name it `hideBorder` so its shorter :D